src/Pages/Main/Components/Slider.js

```diff
@@ -0,0 +1,116 @@
+import React, { useRef, useState, useEffect } from "react";
+import styled from "styled-components";
+import { font, theme } from "../../../Styles/GlobalStyles";
+
+const slideImg = [
+  "/Images/main_images/main_gallery1.png",
+  "/Images/main_images/main_gallery2.png",
+  "/Images/main_images/main_gallery3.png",
+  "/Images/main_images/main_gallery4.png",
+  "/Images/main_images/main_gallery5.png",
+  "/Images/main_images/main_gallery6.png",
+  "/Images/main_images/main_gallery7.webp",
+  "/Images/main_images/main_gallery8.webp",
+];
+
+const TOTAL_SLIDES = 5;
+
+function Slider() {
+  const [currentSlide, setCurrentSlide] = useState(0);
+  const slideRef = useRef(null);
+
+  const nextSlide = () => {
+    if (currentSlide >= TOTAL_SLIDES) {
+      setCurrentSlide(currentSlide + 1);
+    } else {
+      setCurrentSlide(currentSlide + 1);
+    }
+  };
+
+  const prevSlide = () => {
+    if (currentSlide > 0) {
+      setCurrentSlide(currentSlide - 1);
+    } else if (currentSlide === 0) {
+      setCurrentSlide(slideImg.length - 1);
+    }
+  };
```

Comment:
```suggestion
  const prevSlide = () => {
    const moveIdx = currentSlide ? currentSlide - 1 : slideImg.length - 1
    setCurrentSlide(moveIdx)
  };
```
set 함수는 가능한 가장 마지막에 한번 써주세요

Reply:
네! 헐 너무 좋은 방법이네요 3항연산자를 이렇게도 쓸수있구나..